plugins/modules/wafv2_web_acl.py

```diff
@@ -341,8 +413,12 @@ def create(self, default_action, rules, sampled_requests, cloudwatch_metrics, me
                 'SampledRequestsEnabled': sampled_requests,
                 'CloudWatchMetricsEnabled': cloudwatch_metrics,
                 'MetricName': metric_name
-            }
+            },
+            'CustomResponseBodies': custom_response_bodies
```

Comment:
```suggestion
```

breaks backward compatibiliy. Otherwise it is requested all the time, even if not requested

Reply:
ah, probably leftover, as I am adding it conditionally below.